pkg/controllers/gateway/params_test.go

```diff
@@ -8,20 +8,18 @@ import (
 	"reflect"
 	"testing"
 
+	testutil "github.com/Kuadrant/multicluster-gateway-controller/test/util"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/client/fake"
 	gatewayv1beta1 "sigs.k8s.io/gateway-api/apis/v1beta1"
-
-	testutil "github.com/Kuadrant/multicluster-gateway-controller/test/util"
 )
 
 func TestGetParams(t *testing.T) {
 	cases := []struct {
-		name string
-
+		name         string
 		gatewayClass *gatewayv1beta1.GatewayClass
 		paramsObj    client.Object
 		assertParams func(*Params, error) error
```

Comment:
This is neat assertion. Could we reuse it in other assertion functions and move this implementation to the helper package?